### DISCUSSION TITLE
[FEATURE]  This is a suggestions, which may be an improvement, to make all link headings on the How to’s page parallel with one another: either all in the singular or plural

### DISCUSSION THREAD

**Discussion by DhruvPatel2502** [gc-da11yn/gc-da11yn.github.io#565] [Content (Open Discussion)]
### Describe the solution you'd like

All the headings on the How to's pages parallel with one other like this:

-  Host accessible virtual events
- Create forms
- Design courses
- Make emails accessible
- Create documents
- Create web content
- Design accessible services
- Test your products
 
Also, can change “Create forms” to “Create a form”, same for Create document” to “Create a document” and "Design course" to  “Design a course. to make them align with each other

### Who shall do the work?

- [ ] I will do it, I want to contribute
- [x] I am asking for someone else to please do the work

**Comment by shawnthompson** [comment]
@JulieBB23 Thoughts on this request.

**Comment by JulieBB23** [comment]
Go ahead, all good.

**Comment by shawnthompson** [comment]
On this page:

[How to’s - Digital Accessibility Toolkit](https://a11y.canada.ca/en/how-to-s/)

Meaning to make sure all the pages on the site are using the similar grammar:

Should it be "Create document" or just "Create **a** document" if we are using "Design a course"?

**Comment by shawnthompson** [comment]
Should we make all the page to use the grammar?

Suggestion the following changes to this titles:

- [Accessible virtual events](https://a11y.canada.ca/en/accessible-virtual-events/) to **Host accessible virtual events**
- [Create document](https://a11y.canada.ca/en/create-document/) to **Create documents**
- [Create forms](https://a11y.canada.ca/en/create-forms/) no change
- [Create web content](https://a11y.canada.ca/en/create-web-content/) no change
- [Design a course](https://a11y.canada.ca/en/design-a-course/) to **Design courses**
- [Designing accessible services](https://a11y.canada.ca/en/designing-accessible-services/) to **Design accessible services**
- [Making Accessible Emails](https://a11y.canada.ca/en/making-accessible-emails/) to **Create accessible emails**
- [Test your products](https://a11y.canada.ca/en/test-your-products/) no change

We need to make sure the same consistency is done in the French titles on [Comment faire - Boîte à outils de l'accessibilité numérique](https://a11y.canada.ca/fr/comment-faire/index.html).

**Comment by Syracopoulos** [comment]
I like the standardized grammar that you are suggesting @shawnthompson

**Comment by martynka** [comment]
Dido. I would change the "Design courses" to "Design accessible courses", I think that's what the content is about.

**Comment by lynelal** [comment]
I think 'accessible' belong is all the names.  That's our point.

**Comment by martynka** [comment]
Ah, in that case should we remove all the instances of word **accessible** from the list. Or not;)

**Comment by shawnthompson** [comment]
I've converted this into a discussion and once resolved, we can convert is back to an issue with clear instructions on the change that needs to be made.

Go team!

**Comment by Syracopoulos** [comment]
The points that @lynelal and @shawnthompson made in our other discussion is pretty compelling and the standard dictates our answer. We need to have the word accessibility in the heading. Let's just try to not be too repetitive in the description of the page :)

[8:08 AM] Thompson, Shawn (SSC/SPC)
Please keep in mind that WCAG talks about headings being descriptive enough in relation to the content after it. So do you think removing accessibility from the heading on the page describes the content enough for someone to not have to rely on knowing the site is about accessibility?

[8:17 AM] Marjanska, Martyna (SSC/SPC)
I would keep the word "accessibility" in the headings. The individual pages will be coming up in the search on the Web. The context and purpose of the page will be communicated through that heading.
